setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = infobip-api-python-sdk
-version = 3.1.0
+version = 4.0.0
 author = Luka Kilic, Dino Lozina
```

Comment:
Don't forget to add yourself as an author of this, you did the work you should get the credit

Reply:
Done ;)